src/_settings/mappings/career.json

```diff
@@ -0,0 +1,133 @@
+{
+  "properties": {
+    "active": {
+      "type": "boolean"
+    },
+    "applicant_types": {
+      "properties": {
+        "announcement_number": {
+          "type": "string"
+        },
+        "application_type": {
+          "properties": {
+            "active": {
+              "type": "boolean"
+            },
+            "applicant_type": {
```

Comment:
We could change it to either, but you'd still need to access it like `applicant_types.application_type.name`